prometheus-exporters/vrops-exporter/alerts/vccluster.alerts

```diff
@@ -0,0 +1,67 @@
+groups:
+- name: vccluster.alerts
+  rules:
+  - alert: VCenterRedundancyLostFailoverHostMissing
+    expr: |
+      count by (vcenter, vccluster) (vrops_hostsystem_configuration_dasconfig_admissioncontrolpolicy_failoverhost{vccluster=~"production.*"} == 1)
+    for: 30m
+    labels:
+      severity: critical
+      tier: vmware
+      service: compute
+      meta: "Missing failover host for in {{ $labels.vcenter }} cluster {{ $labels.vccluster }}."
+      playbook: docs/devops/alert/vcenter/#restore-ha-redundancy-in-vcenter
+    annotations:
+      description: "Missing failover host for in {{ $labels.vcenter }} cluster {{ $labels.vccluster }}."
+      summary: "Missing failover host for in {{ $labels.vcenter }} cluster {{ $labels.vccluster }}."
+  - alert: VCenterRedundancyLostHAPolicyFaulty
+    expr: |
+      sum by (datacenter, vcenter) (vrops_cluster_configuration_dasconfig_enabled{vccluster=~"production.*"}) unless
+      sum by (datacenter, vcenter) (vrops_hostsystem_configuration_dasconfig_admissioncontrolpolicy_failoverhost{vccluster=~"production.*"}) != 0
+    for: 30m
+    labels:
+      severity: critical
+      tier: vmware
+      service: compute
+      meta: "VC {{ $labels.vcenter }} has a faulty AdmissionControlPolicy for cluster {{ $labels.datacenter }}, failover will not work."
+      playbook: docs/devops/alert/vcenter/#restore-ha-redundancy-in-vcenter
+    annotations:
+      description: "VC {{ $labels.vcenter }} has a faulty AdmissionControlPolicy for cluster {{ $labels.datacenter }}, failover will not work."
+      summary: "VC {{ $labels.vcenter }} has a faulty AdmissionControlPolicy for cluster {{ $labels.datacenter }}, failover will not work."
+  - alert: VCenterRedundancyLostHAPolicyNotConfigured
+    expr: vrops_cluster_configuration_dasconfig_enabled{vccluster=~"production.*"} == 0
+    for: 30m
+    labels:
+      severity: critical
+      tier: vmware
+      service: compute
+      meta: "VC {{ $labels.hostname }} {{ $labels.vccluster }} has a faulty AdmissionControlPolicy. Failover will not work."
+      playbook: docs/devops/alert/vcenter/#restore-ha-redundancy-in-vcenter
+    annotations:
+      description: "VC {{ $labels.hostname }} {{ $labels.vccluster }} has a faulty AdmissionControlPolicy. Failover will not work."
+      summary: "VC {{ $labels.hostname }} {{ $labels.vccluster }} has a faulty AdmissionControlPolicy. Failover will not work."
+  - alert: VCenterWrongHALevelConfiguration
+    expr: |
+      count by (vccluster, vcenter) (vrops_hostsystem_configuration_dasconfig_admissioncontrolpolicy_failoverhost{vccluster=~"production.*"} == 1) > 2
```

Comment:
same as in L6? This time with expression.

Reply:
No, this one should be right.

Reply:
I rechecked the expression and it works as expected.

Reply:
Alright, just didn't add up with L6 in the first place.